src/screens/auth/UpdatePassword.tsx

```
@@ -0,0 +1,147 @@
+
+const UpdatePassword: React.FC = () => {
+    const { t } = useTranslation();
+    const navigate = useNavigate();
+    const toast = useToast();
+
+    const [oldPassword, setOldPassword] = useState('');
+    const [newPassword, setNewPassword] = useState('');
+    const [confirmPassword, setConfirmPassword] = useState('');
+    const [loading, setLoading] = useState(false);
+
+    const username = localStorage.getItem('pendingUser');
```

ESLint

[error] Replace `····` with `↹`

(prettier/prettier)